src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,287 @@
+     * @chainable
+     * @private
+     */
+    _enable() {
+        this._eventBus.on(EVENT.AIRPORT_CHANGE, this.onAirportChangeHandler);
+        this.onAirportChange();
+
+        return this;
+    }
+
+    /**
+     * Disable all event handlers and destroy the instance
+     *
+     * @for AirportInfoController
+     * @method _disable
+     * @chainable
+     * @private
+     */
+    _disable() {
+        this.$element = null;
+        this.$template = null;
+        this.simClockController = null;
+        this.airportInfoController = null;
+
+        this._eventBus.off(EVENT.AIRPORT_CHANGE, this._onAirportChangeHandler);
+
+        return this;
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method reset
+     * @chainable
+     */
+    reset() {
+        this.wind = null;
+        this.altimeter = null;
+        this.elevation = null;
+        this.icao = null;
+
+        return this;
+    }
+
+    /**
+     * Updates the information taken from the AirportModel: the wind, the altimeter,
+     * and the elevation. Triggered on airport change.
+     *
+     * @for AirportInfoController
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        const airport = AirportController.airport_get();
+        const icao = airport.icao.toUpperCase();
+        const windAngle = Math.round(radiansToDegrees(airport.wind.angle));
+
+        this.wind = this._buildWindAndGustReadout({ speed: airport.wind.speed, angle: windAngle });
+        this.altimeter = this._generateAltimeterReading(airport.wind.speed);
+        this.elevation = `${airport.elevation}`;
+        this.icao = icao;
+
+        this._render();
+    }
+
+    /**
+     * Sets the values from the updated airport info.
+     *
+     * @for AirportInfoController
+     * @method _render
+     * @private
+     */
+    _render() {
+        this.$windView.text(`${this.icao} ${this.wind}`);
+        this.$altimeterView.text(`${this.icao} ${this.altimeter}`);
+        this.$elevationView.text(`${this.icao} ${this.elevation}`);
+    }
+
+    /**
+     * Updates the clock, called from `AppController#update_pre`
+     *
+     * @for AirportInfoController
+     * @method updateClock
+     */
+    updateClock() {
+        this.simClockController.update();
+
+        const time = this.simClockController.render();
+
+        this.$clockView.text(time);
+    }
+
+    /**
+     * Formats the wind angle and speed from object into a string,
+     * in the format `${newAngle}${newSpeed}G${gustSpeed}`.
+     *
+     * @for AirportGameInfoView
+     * @method _buildWindAndGustReadout
+     * @param {Object} wind
+     * @returns {String} formatted string
+     * @private
+     */
+    _buildWindAndGustReadout(wind) {
+        const minGustStrength = 5;
+        const speed = wind.speed;
+        const angle = wind.angle;
+        const newAngle = leftPad((angle || 360), 3);
+        const newSpeed = leftPad(speed, 2);
+        // Creates a fake "gusting" speed
```

Comment:
is this gust speed honored anywhere else?  or is it for display purposes only?

Reply:
display only. There is another PR for wind variation, but no idea if it includes gusts. #762